server/images.go

```diff
@@ -546,16 +546,9 @@ func CreateModel(ctx context.Context, name, modelFileDir, quantization string, m
 		}
 	}
 
-	unref := make(map[string]struct{})
-	if manifest, _, err := GetManifest(ParseModelPath(name)); err == nil {
-		for _, layer := range manifest.Layers {
-			if !slices.Contains(digests, layer.Digest) {
-				unref[layer.Digest] = struct{}{}
-			}
-		}
-
-		if manifest.Config.Digest != layer.Digest {
-			unref[manifest.Config.Digest] = struct{}{}
+	if !envconfig.NoPrune {
+		if old, err := ParseNamedManifest(name); err == nil {
+			defer func() { _ = old.RemoveLayers() }()
```

Comment:
```suggestion
			defer old.RemoveLayers()
```

Would this work? The formatting here looks a bit weird.

Reply:
This is to make the linter happy. I'll add  `// nolint: errcheck` instead